bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -541,7 +540,7 @@ public void onComplete(@Nullable Result result) {
      */
     private void handleTouchEvents(TouchEvents touchEvents) {
         touchEvents.getEvents().forEach(event -> {
-            logger.debug("panel: {} gesture: {}", event.getPanelId(), event.getGesture());
+            logger.info("panel: {} gesture id: {}", event.getPanelId(), event.getGesture());
```

Comment:
why did you increase the log-level?

Reply:
Hi J-N-K,
Because it tells that a particular canvas panel which is comparable to a switch device was pressed by someone. It is important to know that this was happening in particular if someone happened to forget adding a discovered panel thing. The actual comparison of that action can be related to an added thing happens below and will only be logged on debug.

Does that help?
Stefan